packages/integrations/cloudflare/README.md

```diff
@@ -236,71 +283,26 @@ declare namespace App {
 }
 ```
 
-### Environment Variables
+## Platform
 
-See Cloudflare's documentation for [working with environment variables](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables).
+### Headers
 
-```js
-// pages/[id].json.js
-
-export function GET({ params }) {
-  // Access environment variables per request inside a function
-  const serverUrl = import.meta.env.SERVER_URL;
-  const result = await fetch(serverUrl + "/user/" + params.id);
-  return {
-    body: await result.text(),
-  };
-}
-```
+You can attach [custom headers](https://developers.cloudflare.com/pages/platform/headers/) to your responses by adding a `_headers` file to your build output directory. This means it should be placed in your Astro project’s `public/` directory.
```

Comment:
```suggestion
You can attach [custom headers](https://developers.cloudflare.com/pages/platform/headers/) to your responses by adding a `_headers` file in your Astro project's `public/` folder. This file will be copied to your build output directory. 
```
I think where you have to put the file is more important than where it ends up, if you're guiding the reader, so for this and the following of this structure,i'd suggest swapping the order something like this.